src/php/Runtime/Runtime.php

```diff
@@ -204,7 +204,8 @@ private function loadCachedFile(string $filename, string $ns): bool
      */
     protected function loadFile(string $filename, string $ns): void
     {
-        $code = $this->compilerFacade->compile($filename);
+        $content = file_get_contents($filename);
+        $code = $this->compilerFacade->compile($content, $filename);
```

Comment:
Makes sense to pass the `filename` in both methods?

Reply:
Yes. The `compiler()' second argument is the source of the content, it's something required/needed by the lexer in order to identify the source of such, but it's independent of the content itself. This way, the source it's actually an optional argument (being `string` the default value).